cnxarchive/views.py

```diff
@@ -663,11 +663,14 @@ def in_book_search(request):
 
     settings = get_current_registry().settings
     connection_string = settings[config.CONNECTION_STRING]
-    statement = SQL['get-in-book-search']
+    statement = SQL['get-in-collated-book-search']
     with psycopg2.connect(connection_string) as db_connection:
         with db_connection.cursor() as cursor:
             cursor.execute(statement, args)
             res = cursor.fetchall()
+            if not res:  # Perhaps it's not collated
```

Comment:
TBH, I'm not 100% comfy with this idiom here, since there may be other reasons that it's not returning a result. Perhaps I should have gone with the 'do an extra  round trip to the DB to test for is_collated, then call the right one' here.